packages/web-components/src/theme/design-tokens.ts

```diff
@@ -1,388 +1,384 @@
+export const colorPaletteGrapeBackground2 = '--colorPaletteGrapeBackground2';
+export const colorPaletteGrapeForeground2 = '--colorPaletteGrapeForeground2';
+export const colorPaletteGrapeBorderActive = '--colorPaletteGrapeBorderActive';
+export const colorPaletteLilacBackground2 = '--colorPaletteLilacBackground2';
+export const colorPaletteLilacForeground2 = '--colorPaletteLilacForeground2';
+export const colorPaletteLilacBorderActive = '--colorPaletteLilacBorderActive';
+export const colorPalettePinkBackground2 = '--colorPalettePinkBackground2';
+export const colorPalettePinkForeground2 = '--colorPalettePinkForeground2';
+export const colorPalettePinkBorderActive = '--colorPalettePinkBorderActive';
+export const colorPaletteMagentaBackground2 = '--colorPaletteMagentaBackground2';
+export const colorPaletteMagentaForeground2 = '--colorPaletteMagentaForeground2';
+export const colorPaletteMagentaBorderActive = '--colorPaletteMagentaBorderActive';
+export const colorPalettePlumBackground2 = '--colorPalettePlumBackground2';
+export const colorPalettePlumForeground2 = '--colorPalettePlumForeground2';
+export const colorPalettePlumBorderActive = '--colorPalettePlumBorderActive';
+export const colorPaletteBeigeBackground2 = '--colorPaletteBeigeBackground2';
+export const colorPaletteBeigeForeground2 = '--colorPaletteBeigeForeground2';
+export const colorPaletteBeigeBorderActive = '--colorPaletteBeigeBorderActive';
+export const colorPaletteMinkBackground2 = '--colorPaletteMinkBackground2';
+export const colorPaletteMinkForeground2 = '--colorPaletteMinkForeground2';
+export const colorPaletteMinkBorderActive = '--colorPaletteMinkBorderActive';
+export const colorPalettePlatinumBackground2 = '--colorPalettePlatinumBackground2';
+export const colorPalettePlatinumForeground2 = '--colorPalettePlatinumForeground2';
+export const colorPalettePlatinumBorderActive = '--colorPalettePlatinumBorderActive';
+export const colorPaletteAnchorBackground2 = '--colorPaletteAnchorBackground2';
+export const colorPaletteAnchorForeground2 = '--colorPaletteAnchorForeground2';
+export const colorPaletteAnchorBorderActive = '--colorPaletteAnchorBorderActive';
+export const colorPaletteRedForegroundInverted = '--colorPaletteRedForegroundInverted';
+export const colorPaletteGreenForegroundInverted = '--colorPaletteGreenForegroundInverted';
+export const colorPaletteYellowForegroundInverted = '--colorPaletteYellowForegroundInverted';
+export const shadow2 = '--shadow2';
+export const shadow4 = '--shadow4';
+export const shadow8 = '--shadow8';
+export const shadow16 = '--shadow16';
+export const shadow28 = '--shadow28';
+export const shadow64 = '--shadow64';
+export const shadow2Brand = '--shadow2Brand';
+export const shadow4Brand = '--shadow4Brand';
+export const shadow8Brand = '--shadow8Brand';
+export const shadow16Brand = '--shadow16Brand';
+export const shadow28Brand = '--shadow28Brand';
+export const shadow64Brand = '--shadow64Brand';
```

Comment:
Let's discuss taking a common dep on @fluentui/tokens. We're nearly identical between this and that now:

https://github.com/microsoft/fluentui/blob/master/packages/tokens/src/tokens.ts

Reply:
Yeah, I'm fine with generating this locally at build time for instance - I didn't want to block the improvement on that.

Reply:
At least there should be a test to verify the tokens here match the tokens in `@fluentui/tokens`